addon/components/variable-number/number.ts

```diff
@@ -44,7 +44,20 @@ export default class VariableNumberPluginNumberComponent extends Component<Args>
     return this.args.node;
   }
   get formattedNumber() {
-    return this.node.attrs.value as string;
+    const writtenNumber = this.writtenNumber;
```

Comment:
I think this is not needed (probably forgot to delete/change it?)
It would make sense to add `const value = this.node.attrs.value as string` to make code more readable though.